.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,52 @@
+---
+name: 🐞 Bug report
+about: Create an issue which describes a bug of CloudNet
+
+---
+
+**Describe the bug**<br>
+<!-- A clear and concise description of what the bug is. Summerize the bug with a few words. -->
+
+
+**Steps for reproducing**<br>
+<!-- Here is a example how this could look like. The import thing is that you are describing the steps as specific as possible! -->
```

Comment:
```suggestion
<!-- Here is a example how this could look like. The important thing is that you are describing the steps as specific as possible! -->
```

Reply:
Gotcha.